py21cmsense/yaml.py

```diff
@@ -39,7 +45,18 @@ def wrapper(data):
                 raise LoadError(str(e))
 
         def yaml_fnc(loader, node):
-            return wrapper(node.value)
+            args = node.value.split("|")
+            if len(args) == 1:
+                return wrapper(node.value)
+            elif len(args) == 2:
+                # Return with a unit
+                return wrapper(args[0].strip()) * getattr(un, args[1].strip())
```

Comment:
Are there any units we use we might expect not to be simply'ed `getattr`-able? Astropy does support some more complex units via the Quantity(foo, unit='bar') interface. Just want to make sure we're going to be covering as many cases as possible.

Reply:
Yeah, I did think about that. I think in our case, it'll be 90% "m". In fact I can't think of another use-case off the top of my head. Also if it pops up I could add that functionality.

Reply:
that's fair! Figure it would probably be very simple units just wanted to make sure.